apps/vx-scan/backend/src/server.ts

```diff
@@ -0,0 +1,61 @@
+import express, { Application, RequestHandler, response } from 'express';
+import grout from '@votingworks/grout';
+
+export interface Config {
+  electionName: string;
+  precinctName: string;
+}
+
+let store: { config: Config | undefined } = {
+  config: {
+    electionName: 'Election Name',
+    precinctName: 'Precinct Name',
+  },
+};
+
+export type ScannerStatus = 'no_paper' | 'scanning' | 'accepted';
+
+const scannerStateMachine = {
+  state: 'no_paper' as ScannerStatus,
+  transition() {
+    switch (this.state) {
+      case 'no_paper':
+        this.state = 'scanning';
+        break;
+      case 'scanning':
+        this.state = 'accepted';
+        break;
+      case 'accepted':
+        this.state = 'no_paper';
+        break;
+    }
+  },
+};
+
+const api = grout.createApi({
+  getConfig: grout.query(async (): Promise<Config | undefined> => {
+    return store.config;
+  }),
+
+  updateConfig: grout.mutation(async (newConfig: Config): Promise<void> => {
```

Comment:
can we add `mutationWithSideEffect` or something like it that maps to POST? I'm thinking of something like `printBallot`, that isn't idempotent, vs. `saveState` which is.